src/jdk.incubator.jextract/share/classes/jdk/internal/clang/libclang/RuntimeHelper.java

```diff
@@ -46,17 +49,19 @@
 import java.util.stream.Stream;
 
 import static jdk.incubator.foreign.CLinker.*;
+import static jdk.internal.clang.libclang.CLayouts.C_DOUBLE;
+import static jdk.internal.clang.libclang.CLayouts.C_LONG_LONG;
+import static jdk.internal.clang.libclang.CLayouts.C_POINTER;
 
```

Comment:
I assume this is transition code. RuntimeHelper is generated. It cannot use handwritten CLayouts interface (unless we plan to generate something similar from jextract every time)

Reply:
Yep, this is transitional code - jextract will generate these layouts now.